src/engine/strat_engine/backstore/metadata/mda.rs

```diff
@@ -155,7 +155,11 @@ pub struct MDARegions {
     /// The MDA headers which contain information about the variable
     /// length metadata. NUM_PRIMARY_MDA_REGIONS is 2: in the general
     /// case one is more recently written than the other.
-    mdas: [Option<MDAHeader>; mda_size::NUM_PRIMARY_MDA_REGIONS],
+    /// A value of None indicates that no variable length metadata has been
+    /// written to the MDA regions corresponding to a given MDA header.
+    /// If there is Some value, then variable length metadata has been read;
+    /// the MDA header's used attribute therefore can not be 0 bytes.
```

Comment:
"\`used\` field" instead maybe?

Reply:
Yes, field is probably better.

Reply:
> On line 160 can the word some have a lower case s?

@drckeefe: That's a Rust-specific "Some", it's one of the two constructors of the Option type.
https://doc.rust-lang.org/std/option/enum.Option.html. That's short for saying something like "if the value has been constructed with the Some constructor", or something like that.

Reply:
Not a constructor, a variant. (sorry for nitpicking) :)

Reply:
It is both! But that would be somewhat easier to say: "If the value is the Some variant of the Option type".